src/CritCore/Config/config.js

```diff
@@ -1,13 +1,28 @@
 const config = {
-  language: 'en',
-  defaultLanguage: 'en',
   about: {
     docsUrl: 'https://docs.google.com/document/d/1-o3B3WHAnCwsz80m3C76dEQzB4DJ3NoNrvJGZrzN6gc/edit?usp=sharing'
   },
+  defaultLanguage: 'en',
+  language: 'en',
+  passwordSettings: {
```

Comment:
Rename this key to registerFormSettings.

Reply:
Done.